bedita-app/models/behaviors/build_filter.php

```diff
@@ -1208,9 +1209,9 @@ protected function allowed_to_userFilter($s, $e, $userid) {
                     $this->conditions[] = "(
                         (
                             NOT EXISTS (
-                                SELECT {$s}Tree{$e}.{$s}id{$e}
+                                SELECT DISTINCT {$s}Tree{$e}.{$s}id{$e}
                                 FROM {$s}trees{$e} AS {$s}Tree{$e}
-                                WHERE {$s}Tree{$e}.{$s}id{$e} = {$s}BEObject{$e}.{$s}id{$e}
+                                INNER JOIN {$s}objects{$e} AS {$s}BEObject{$e} ON {$s}BEObject{$e}.{$s}id{$e} = {$s}Tree{$e}.{$s}id{$e}
```

Comment:
This changes the query meaning… it becomes a non-dependent subquery, which checks if _any_ object is on the tree. It is a condition that is basically never satisfied.

Reply:
I think you're right :disappointed: